### PR TITLE
[Internal review] Million-scale Face SDK generated with local swagger branch and related tests.

### PR DESCRIPTION
**Pull request by lebronJ**
Updated by local MillionScale Swagger branch.
After getting this reviewed, will first get the Swagger PR merged to [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) and then generate SDK from its master branch.

### CODE REVIEW

**Comment by lebronJ** [comment]
@huxuan, this is for internal review.

**Comment by huxuan** [comment]
Gotcha. Will do it today.

File: src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face.Tests/FaceVerificationTests.cs [hunk 1]
```diff
@@ -40,11 +40,89 @@ public void FaceVerificationPositive()
...
+                client.LargePersonGroup.CreateAsync(largePersonGroupId, "fakeLargePersonGroup").Wait();
+                try
+                {
+                    Person createPersonResult = client.LargePersonGroupPerson.CreateAsync(largePersonGroupId, "David").Result;
+                    using (FileStream stream = new FileStream(Path.Combine("TestImages", "verificationBase1.png"), FileMode.Open))
+                    {
+                        client.LargePersonGroupPerson.AddPersonFaceFromStreamAsync(largePersonGroupId, createPersonResult.PersonId, stream).Wait();
+                    }
+
+                    client.LargePersonGroup.TrainAsync(largePersonGroupId).Wait();
```
Comment (huxuan) [inline src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face.Tests/FaceVerificationTests.cs]: Wait for training here? Same for other similar places.

**Comment by huxuan** [comment]
The overall looks good. Please make sure it is compatible with both of the old and new services. The only concern is about the wait for training which may be misleading to new comers/customers.

File: src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face.Tests/FaceVerificationTests.cs (hunk 1 above)
Comment (lebronJ) [reply src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face.Tests/FaceVerificationTests.cs]: Yes, added in latest commit.

**Comment by lebronJ** [comment]
@huxuan Yes. Confirmed this SDK works with our new service as well, in terms of functionalities regardless of faceList/personGroup.

File: src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face.Tests/BaseTests.cs [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
         public static bool IsTestTenant = false;
         private static readonly string FaceSubscriptionKey;
-        
+
         static BaseTests()
         {
             // Retrieve the configuration information.
-            FaceSubscriptionKey = "";
+            //Environment.SetEnvironmentVariable("AZURE_TEST_MODE", "Record");
+            //FaceSubscriptionKey = "793315696cd04d5ab78232a792136610";
```
Comment (huxuan) [inline src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face.Tests/BaseTests.cs]: Maybe we should remove the actual value for `FaceSubscriptionKey` though it is a fake one.

**Review by huxuan — APPROVED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face.Tests/BaseTests.cs (hunk 2 above)
Comment (lebronJ) [reply src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face.Tests/BaseTests.cs]: Yes, this is just for code sharing. Raised formal PR to upstream repo.

**Pull request merged**